packages/wdio-cucumber-framework/src/utils.ts

```diff
@@ -95,7 +115,7 @@ export function buildStepPayload(
 ) {
     return {
         uid: step.id,
-        title: step.text,
+        title: createStepTitle(step),
```

Comment:
Let's safe one more function call

```suggestion
        title: isStepTypeHook(step) ? `hook-${step.hookId}` : step.text,
```

Reply:
Thanks, I will apply this.